project-icp/bee-pollinator-app
# Export all survey tables to CSV

## Overview

Site admins want the data as CSV to do analyses. This PR adds a view at http://localhost:8000/beekeepers/export for admins to download tables as a zip of CSVs. SQL queries are hardcoded.

Connects #384
Connects #385

### Demo

The idea: (i fussed with the filename timestamp a little since this screenshot)
<img width="1093" alt="screen shot 2019-01-22 at 11 13 20 am" src="https://user-images.githubusercontent.com/10568752/51562415-c9509400-1e57-11e9-94a8-1a76d5a28ff9.png">

The date formatter doesn't allow colons in the time (only `-`). It's livable. It's main purpose is to give the files a unique ID.
<img width="806" alt="screen shot 2019-01-22 at 11 25 34 am" src="https://user-images.githubusercontent.com/10568752/51562417-c9509400-1e57-11e9-9821-443a9962648c.png">
Non-admins can't access:
<img width="1014" alt="screen shot 2019-01-22 at 11 14 42 am" src="https://user-images.githubusercontent.com/10568752/51562416-c9509400-1e57-11e9-8ff1-0976870680f8.png">
<img width="541" alt="screen shot 2019-01-22 at 3 14 22 pm" src="https://user-images.githubusercontent.com/10568752/51562701-70353000-1e58-11e9-9310-b0e26566b497.png">


## Testing Instructions

Visit the URL and sign in as an admin. Check that the zip downloads and the CSVs look good.
